proxy/src/transparency/server.rs

```diff
@@ -148,7 +158,8 @@ where
         let tcp = self.tcp.clone();
         let new_service = self.new_service.clone();
         let drain_signal = self.drain_signal.clone();
-        Either::A(io.peek()
+        let ctx = ServeInfo(srv_ctx.clone());
```

Comment:
`ctx` is kind of close to `srv_ctx`, which has a totally different concept. Maybe `log_ctx` or something?